include/actuator/ActuatorTypes.h

```diff
@@ -16,9 +16,9 @@
 namespace sierra {
 namespace nalu {
 
-#ifdef ACTUATOR_ON_DEVICE
+#ifdef __CUDACC__
```

Comment:
We have been using `KOKKOS_ENABLE_CUDA` in other places, can you switch to that? Will be easier to replace it with whatever Kokkos suggests for `KOKKOS_ENABLE_<ACCELERATOR>`

Reply:
Sure thing.